### ISSUE TITLE
Add a way to test if element is in view?

### ISSUE THREAD

**Issue by drewbaker** [funkhaus/whenInView#6]
Would be great if this plugin also allowed something like this to work: `jQuery('#el').is(':inview')`

**Comment by johndigital** [comment]
Would this work for what you need? `jQuery('#el').hasClass('element-in-view')`

That's using the default class, so if you had a custom one you would just have to change it. Having a custom pseudo-selector like you mentioned would be pretty cool though too.

**Comment by drewbaker** [comment]
Yeah that class test works fine, but that pseudo class way would be sweet.
